src/pages/SignIn/components/UserSignIn.jsx

```diff
@@ -20,19 +21,28 @@ const UserSignIn = () => {
 
   return (
     <div className="sign-in-container">
-      <h1>Sign In</h1>
       <form id="myForm" onSubmit={signIn}>
```

Comment:
i think you don't need form here and move signIn function to Button OnClick?

Reply:
then you can remove `e.preventDefault();` too

Reply:
e.preventDefault is used to handle form submission and prevent page refresh. without it, when the user clicks "continue with email," page just refreshes without navigating to profile page.